src/main/io/osd.c

```diff
@@ -1180,6 +1180,17 @@ static bool osdDrawSingleElement(uint8_t item)
             break;
         }
 
+    case OSD_HOME_HEADING_ERROR:
+        {
+            buff[0] = SYM_HOME;
+            buff[1] = SYM_HEADING;
+            int16_t h = wrap_18000(DEGREES_TO_CENTIDEGREES((int32_t)GPS_directionToHome) - DECIDEGREES_TO_CENTIDEGREES((int32_t)osdGetHeading()));
```

Comment:
Why is this in 0.1deg? Isn't 1deg precise enough? MWOSD seems to be displayed full degrees https://github.com/ShikOfTheRa/scarab-osd/blob/2f2311363a95ad797dbb92621330b6c2ffab8e79/MW_OSD/Screen.ino#L1191

Reply:
Wouldn't  `osdGetHeadingAngle()` work for the wrapping?

Reply:
Ah, scratch that. I got it, you want CCW to be negative. Makes sense that way.